docs/tarball.md

```diff
@@ -0,0 +1,170 @@
+| percona-patroni | A high-availability solution for PostgreSQL |
+| percona-pgbackrest| A backup and restore tool |
+| percona-pgbadger| PostgreSQL log analyzer with fully detailed reports and graphs |
+| percona-pgbouncer| Lightweight connection pooler for PostgreSQL |
+| percona-pgpool-II| A middleware between PostgreSQL server and client for high availability, connection pooling and load balancing |
+| percona-perl | A Perl module required to create the `plperl` extension - a procedural language handler for PostgreSQL that allows writing functions in the Perl programming language|
+| percona-python3 | A Python3 module required to create `plpython` extension - a procedural language handler for PostgreSQL that allows writing functions in the Python programming language. Python is also required by Patroni
+| percona-tcl | Tcl development libraries required to create the `pltcl` extension - a loadable procedural language for the PostgreSQL database system that enables the creation of functions and trigger procedures in the Tcl language |
+| percona-etcd | A key-value distributed store that stores the state of the PostgreSQL cluster|
+
+## Preconditions
+
+=== "Debian and Ubuntu"
+
+    1. Uninstall the upstream PostgreSQL package. 
+    2. Create the user to own the PostgreSQL process. For example, `mypguser`. Run the following command:
+
+        ```{.bash data-prompt="$"}
+        $ sudo useradd -m mypguser
+        ```
+
+        Set the password for the user:
+
+        ```{.bash data-prompt="$"}
+        $ sudo passwd mypguser
+        ```
+    
+=== "RHEL and derivatives"
+
+    Create the user to own the PostgreSQL process. For example, `mypguser`, Run the following command: 
+        
+    ```{.bash data-prompt="$"}
+    $ sudo useradd mypguser -m 
+    ```
+
+    Set the password for the user:
+
+    ```{.bash data-prompt="$"}
+    $ sudo passwd mypguser
+    ```
+
+## Procedure
+
+The steps below install the tarballs for OpenSSL 3.x. Use another tarball if your operating system has OpenSSL version 1.x.
+
+1. Create the directory where you will store the binaries. For example, `/opt/pgdistro`
+
+2. Grant access to this directory for the `mypguser` user.
+
+    ```{.bash data-prompt="$"}
+    $ sudo chown mypguser:mypguser /opt/pgdistro/
+    ```
+
+3. Fetch the binary tarball:
+
+    ```{.bash data-prompt="$"}
+    $ wget https://downloads.percona.com/downloads/postgresql-distribution-{{pgversion}}/{{dockertag}}/binary/tarball/percona-postgresql-{{dockertag}}-ssl3-linux-x86_64.tar.gz
+    ```
+
+4. Extract the tarball to the directory for binaries that you created on step 1.
+
+    ```{.bash data-prompt="$"}
+    $ sudo tar -xfv percona-postgresql-{{dockertag}}-ssl3-linux-x86_64.tar.gz -C /opt/pgdistro/
+    ```
+
+5. If you extracted the tarball somewhere else in your system, copy `percona-python3`, `percona-tcl` and `percona-perl` to the `/opt` directory. This is required for the correct run of libraries that require those modules. 
```

Comment:
I would suggest write it as follows:

If you have extracted the tarball in a directory other than /opt, copy `percona-python3`, `percona-tcl` and `percona-perl` to the `/opt` directory. This is required for the correct run of libraries that require those modules.